srcpkgs/ponyc/template

```diff
@@ -1,21 +1,22 @@
 # Template file for 'ponyc'
 pkgname=ponyc
-version=0.20.0
+version=0.24.0
 revision=1
 only_for_archs="x86_64"
 build_style=gnu-makefile
 hostmakedepends="llvm3.9"
 makedepends="zlib-devel ncurses-devel libatomic-devel"
 short_desc="OO, actor-model, capabilities-secure, high-performance language"
-maintainer="bougyman <bougyman@voidlinux.eu>"
-license="2-clause-BSD"
+maintainer="Brian Mitchell <brian@strmpnk.co>"
+license="BSD-2-Clause"
 homepage="http://ponylang.org/"
```

Comment:
https://

Reply:
I tried to leave it untouched but that's a reasonable change. I'll push that shortly.